src/odoc/etc/odoc.css

```
@@ -451,11 +451,13 @@ div.odoc-spec,.odoc-include {
 
 .spec.type .record > .def-doc, .spec.type .variant > .def-doc {
   min-width:50%;
-  padding-left: 22px;
+  padding-left: 18px;
```

If you remove the blue border, a smaller padding would be better: (the same as `.spec`)
```suggestion
  padding: 0.35em 0.5em;
```